### PR TITLE
chore: add prettier workflow

### PR DESCRIPTION
**Pull request by pg56714**

### CODE REVIEW

File: package.json
Comment (stilt0n) [inline package.json:2]: I don't think we need to change the name here.

File: .prettierrc
Comment (stilt0n) [inline .prettierrc:2]: Can we keep the formatting as it was here?

File: .prettierignore
Comment (stilt0n) [inline .prettierignore:6]: Good call modifying the prettier ignore.

Personally, I'm okay with also formatting markdown files. But I'm also okay not doing it.

File: .github/workflows/post-merge.yml
Comment (stilt0n) [inline .github/workflows/post-merge.yml:62]: The reason that this is all in one file is to avoid race conditions:

If one job pushes changes to the repo and then another job attempts later you can run into errors about merge conflicts or the branch being stale.

I think separating the jobs can be nice, so if you really wanted you could create a format job that gets triggered on `workflow_call`. But I don't think you should try to run two jobs in parallel here as it could cause one of the jobs to fail.

My personal preference here is to keep everything in one job since running prettier doesn't seem like a complex enough step that splitting it out into a separate workflow really reduces code complexity.

But I could see either approach as a valid one.

File: package.json
Comment (pg56714) [reply package.json:2]: I understand your point, but the current name does not conform to the format required by npm for package names. According to the rules, the name must match the regex '^(?:(?:@(?:[a-z0-9-~][a-z0-9-.~]*)?/[a-z0-9-.])|[a-z0-9-])[a-z0-9-._~]*$'. This means we need to change the uppercase letters to lowercase to meet these requirements.

File: .prettierrc
Comment (pg56714) [reply .prettierrc:2]: ok

File: .github/workflows/post-merge.yml
Comment (pg56714) [reply .github/workflows/post-merge.yml:62]: done

**Comment by pg56714** [comment]
@stilt0n Everything has been resolved, please review again, thank you.

File: .prettierignore
Comment (pg56714) [reply .prettierignore]: done

**Comment by pg56714** [comment]
> @stilt0n Everything has been resolved, please review again, thank you.

@LaurelineP done

File: .github/workflows/post-merge.yml
Comment (stilt0n) [inline .github/workflows/post-merge.yml:78]: nit: I think it might be nice to keep the commit message format more consistent with the others. i.e. maybe this could be "(CI) Run Prettier"?

**Review by stilt0n — CHANGES_REQUESTED** [review]
This looks pretty good to me now. I have suggested a couple of small changes but like the overall direction.

I also noticed that it looks like you're using a separate push for formatting so that it runs even when the cards don't change. I think that was a good decision.

After you make the changes I will approve this PR. I will probably leave it to @MattCSmith  to make the final decision on merging it.

File: .github/workflows/post-merge.yml
Comment (pg56714) [reply .github/workflows/post-merge.yml]: done

File: .github/workflows/post-merge.yml
Comment (pg56714) [reply .github/workflows/post-merge.yml]: done

**Review by stilt0n — APPROVED** [review]
Thanks for making the changes. This looks good to me. But like I said above, I will wait on Matt's input to merge this one.

**Comment by pg56714** [comment]
Hi @MattCSmith, I would greatly appreciate it if you could take some time to review the changes. Thank you!

**Review by MattCSmith — APPROVED** [review]

**Pull request merged**
